transaction/src/manifest/lexer.rs

```diff
@@ -3,9 +3,9 @@ use sbor::rust::str::FromStr;
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct Span {
     /// The start of the span, inclusive
```

Comment:
As mentioned in the thread, can we document _what_ the index is?

In this case, it's the index in terms of rust characters - ie  Unicode code points / _UTF-32 characters_.